docs/rfcs/1002-merging-process.md

```diff
@@ -158,53 +157,62 @@ Also, for *draft* pull requests:
 - Did you follow the checklist ?
   > A simple spell, but quite unbreakable !
 
-## How do we review a Pull-Request
+## Reviewing a pull request
+
+Before a pull request can be merged, it need to be reviewed to meet our quality
+standards.
+
+The review consists in two steps:
+
+- [The automated review](#the-automated-review)
+- [The peer review](#the-peer-review)
 
-When a change is proposed by a Pull-Request we need to review it to ensure that it meet our quality standard.
+Also, there are some extra steps for [specific pull requests reviews](#specific-pull-request-reviews).
 
-For that, in this section, we will define a review process that can be split into 2 part:
+### The automated review
 
-- [The automated review process.](#the-automated-review-process)
-- [The manual / human review process.](#the-manual-or-human-review-process)
-- [Specific process for specific PR](#specific-process-for-specific-pr)
+The goal is to automate the systematic and tedious checks that can be easily
+forgotten by a peer.
 
-### The automated review process
+This step is responsible for:
 
-The automated part role is to review the tedious part that can be easily forgotten.
-The automated part is responsible to:
+- Ensuring codebase compiles (or equivalent)
+- Executing tests and verifying their result (cargo test, pytest, ...)
+- Verifying coding style using the configured linters (rustfmt, clippy, ruff,
+  eslint, ...)
+- Checking spell issues (cspell)
+- Ensuring the app can still be packaged with the proposed change
+- Ensuring commit signatures are valid (gpg signed commit)
+- Verifying simple checks from the [Pull-request checklist](#the-pull-request-checklist)
+- Verifying simple check on commit titles
 
-- The proposed change can compile (or equivalent).
-- Execute the tests and verify their result (cargo test, pytest, ...).
-- Verify the coding style by using the configured linters (rustfmt, clippy, ruff, eslint, ...).
-- Verify typo
-- Check if we're able to package the app with the proposed change.
-- Basic check that we've followed the [Pull-request checklist](#the-pull-request-checklist).
-- Check the commit signature (gpg signed commit) are valid.
-- Basic check on commit title.
+### The peer review
 
-### The manual or human review process
+The goal is not only to ensure quality standards missed in the automated review,
+but also to encourage **team collaboration** and **knowledge sharing**.
 
-This is the part where one or more person are involved.
-This role of this part is:
+Ideally, the review should be done by the [code owner]((#the-code-owner)) of the
+impacted code parts.
```

Comment:
@mmmarcos 

Note: If we enable the code owner system, It would be a requirement ;)

Reply:
Good proposal, I would like to add that anyone is free to do a review, but it's the code owner one that will allow to merge